runtime/ops/fs.rs

```diff
@@ -1409,10 +1450,12 @@ fn op_symlink_sync(
 #[op]
 async fn op_symlink_async(
   state: Rc<RefCell<OpState>>,
-  args: SymlinkArgs,
+  oldpath: String,
+  newpath: String,
+  _type: Option<String>,
```

Comment:
Nitpick: at first I thought it's an unused variable. Maybe use `type_` or `kind`?

Reply:
or `r#type`... no idea how widely used raw identifiers are though

Reply:
It is indeed an unused variable on Unix. `_type` is to avoid the warning.

Reply:
IMO, when I'd come back through and read this, it would make more sense to see
```rust
let _ = r#type;
```
within the unix block.